tests/PSRule.Rules.Azure.Tests/Azure.AKS.Tests.ps1

```diff
@@ -230,6 +230,22 @@ Describe 'Azure.AKS' -Tag AKS {
             $ruleResult.TargetName | Should -BeIn 'cluster-F';
         }
 
+        It 'Azure.AKS.AutoScaling' {
+            $filteredResult = $result | Where-Object { $_.RuleName -eq 'Azure.AKS.AutoScaling' };
+
+            # Fail
+            $ruleResult = @($filteredResult | Where-Object { $_.Outcome -eq 'Fail' });
+            $ruleResult | Should -Not -BeNullOrEmpty;
+            $ruleResult | Should -HaveCount 4
+            $ruleResult.TargetName | Should -BeIn 'cluster-A', 'cluster-B', 'cluster-C', 'cluster-D';
```

Comment:
In think cluster A, B should be returning Pass. Altnatively you could filter out clusters/ node pools that have no scale set node pools by using a `-If` pre-condition.

Reply:
Fixed